### DISCUSSION TITLE
Anyone up for testing another new Pocketbase hosting service?

### DISCUSSION THREAD

**Discussion by burggraf** [pocketbase/pocketbase#4297] [Show and tell]
Features:
- set up a new project in a few seconds
- host in region(s) around the world
- set up read-only or read-write replicas (marmot)
- streaming backup with point-in-time-recovery (litestream)
- high availability (currently using round robin with automatic failover)
- project cloning (copy prod to staging, dev, etc.)
- and more...
https://discord.gg/aGrUaneA

**Comment by brunowilliang** [comment]
Hi @burggraf, the link has expired

**Comment by eqoram** [comment]
Also very interested in this. :)

**Reply by burggraf** [reply]
@Brunowilliang @eqoram I'll send a new invite later this week.

**Reply by brunowilliang** [reply]
Hey @burggraf Was it ok to send the invitation? because I didn't receive anything, looking forward to testing

**Reply by burggraf** [reply]
@Brunowilliang @eqoram I'm cleaning up a few things before sending it, but it's still coming.

**Reply by burggraf** [reply]
@Brunowilliang @eqoram 
https://discord.gg/nvkpPGfqBh
